1.17 should work

sschr15 opened the pull request:
java 16 is less pain than i imagined

SilverAndro committed:
Set field accessable to make transformerApi actually work

idk bout java 9 who cares about that tho

sschr15 committed:
porting to 1.17 was easier than i imagined

sschr15 committed:
clean up build.gradle a bit

sschr15 committed:
whoops people shouldn't need to have extra ram :tiny_potato:

sschr15 commented:
i already tested transformer api and it works as i'd hope

Devan-Kerman commented:
this is kinda ded, I plan on at some point within the next century modularizing gfh but until then you're prolly better of publishing a fork

sschr15 committed:
fighting with knot
apparently KnotClassLoader as it was before is now KnotCompatibilityClassLoader
so I had to swap that around
then I had to "enforce" loading using KnotCompatibilityClassLoader
also I changed things so it instead just rethrows exceptions to avoid adding an extra unneeded stacktrace cause

sschr15 committed:
whoops still need this

sschr15 committed:
oh did i mention gradle is dumb

sschr15 committed:
readme go brr
